go.mod

```diff
@@ -2,6 +2,14 @@ module github.com/SwissDataScienceCenter/renku-frontend-buildpacks
 
 go 1.24.1
 
+tool (
+	github.com/golangci/golangci-lint/v2/cmd/golangci-lint
```

Comment:
Except `golangci-lint`! 😅 
https://golangci-lint.run/docs/welcome/install/#install-from-sources

Note: I am not a fan of them not suggesting any way to manage `golangci-lint` updates, e.g. with dependabot.